client/blocks/disconnect-jetpack/index.jsx

```diff
@@ -0,0 +1,106 @@
+	}
+
+	planFeatures() {
+		const { plan, translate } = this.props;
+		const features = [];
+		switch ( plan ) {
+			case 'free':
+				features.push(
+					translate( '{{icon/}} Site stats, related content, and sharing tools',
+					this.translateArgs( 'stats-alt' )
+				) );
+				features.push(
+					translate( '{{icon/}} Brute force attack protection and uptime monitoring',
+					this.translateArgs( 'lock' )
+				) );
+				features.push( translate( '{{icon/}} Unlimited, high-speed image hosting', this.translateArgs( 'image' ) ) );
+				break;
+
+			case 'personal':
+				features.push( translate( '{{icon/}} Daily, automated backups (unlimited storage)', this.translateArgs( 'history' ) ) );
+				features.push( translate( '{{icon/}} Priority WordPress and security support', this.translateArgs( 'chat' ) ) );
+				features.push( translate( '{{icon/}} Spam filtering', this.translateArgs( 'spam' ) ) );
+				break;
+
+			case 'premium':
+				features.push( translate( '{{icon/}} Daily, automated backups (unlimited storage)', this.translateArgs( 'history' ) ) );
+				features.push( translate( '{{icon/}} Daily, automated malware scanning', this.translateArgs( 'spam' ) ) );
+				features.push( translate( '{{icon/}} Priority WordPress and security support', this.translateArgs( 'chat' ) ) );
+				features.push( translate( '{{icon/}} 13Gb of high-speed video hosting', this.translateArgs( 'video' ) ) );
+				break;
+
+			case 'professional':
+				features.push( translate( '{{icon/}} Daily, automated backups (unlimited storage)', this.translateArgs( 'history' ) ) );
+				features.push(
+					translate( '{{icon/}} Daily, automated malware scanning with automated resolution',
+					this.translateArgs( 'spam' )
+				) );
+				features.push( translate( '{{icon/}} Priority WordPress and security support', this.translateArgs( 'chat' ) ) );
+				features.push( translate( '{{icon/}} Unlimited high-speed video hosting', this.translateArgs( 'video' ) ) );
+				features.push( translate( '{{icon/}} SEO preview tools', this.translateArgs( 'globe' ) ) );
+				break;
+		}
+
+		return features.map( ( freature, index ) => {
+			return <div key={ 'disconnect-jetpack__feature-' + index } className="disconnect-jetpack__feature">{ freature } </div>;
+		} );
+	}
+	render() {
+		const { onStay, onDisconnect, isVisible, translate, isBroken } = this.props;
+		if ( isBroken ) {
+			return (
+				<Dialog isVisible={ isVisible } baseClassName="disconnect-jetpack__dialog" >
+				<h1>{ translate( 'Disconnect Jetpack' ) }</h1>
+				<p className="disconnect-jetpack__highlight">{ translate( 'WordPress.com has not been able to react example.com for a while.' ) }</p>
+				<div className="disconnect-jetpack__button-wrap">
+					<Button primary scary onClick={ onDisconnect }>{ translate( 'Remove Site' ) }</Button>
+				</div>
+			</Dialog>
+			);
+		}
+
+		return (
+			<Dialog isVisible={ isVisible } baseClassName="disconnect-jetpack__dialog" >
+				<h1>{ translate( 'Disconnect Jetpack?' ) }</h1>
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Disconnect Jetpack' )` *ES Score: 19*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/disconnect-jetpack-block)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).